controllers/installer_controller.go

```diff
@@ -69,37 +123,131 @@ func (r *InstallerReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 	if len(installer.Spec.URL) == 0 {
 		return reconcile.Result{}, fmt.Errorf("url is not specified")
 	}
+	devtronVersion := strings.ReplaceAll(installer.Spec.URL, "https://raw.githubusercontent.com/devtron-labs/devtron/", "")
+	devtronVersion = strings.ReplaceAll(devtronVersion, "/manifests/installation-script", "")
+	if len(devtronVersion) > 8 {
+		devtronVersion = "v1"
```

Comment:
cant understand this logic.. whats the idea? will this same verion be used by orchestrator?

Reply:
removed devtron version for now.